server/viame_tasks/tasks.py

```diff
@@ -150,3 +167,50 @@ def convert_video(self, path, folderId, token, auxiliaryFolderId):
         _file["itemId"], {"folderId": folderId, "codec": "h264"}
     )
     os.remove(output_path)
+
+
+@app.task(bind=True)
+def convert_images(self, folderId, token):
```

Comment:
don't pass `token` as a job arg, use the `girder_client_token` special kwarg.  https://github.com/girder/girder_worker/blob/master/HACKING.md#girder-client

Reply:
Good to know. I took that from other tasks, so I'll fix this one and make an issue for the others.